src/e2e/pages/purchasePage.ts

```diff
@@ -82,8 +82,7 @@ export class PurchasePage {
     await this.paypalButton.click();
     const newPage = await pagePromise;
     await newPage.waitForLoadState();
-    await newPage.waitForURL(/.*paypal\.com.*\/checkout.*/);
-
+    await newPage.waitForURL(/.*paypal\.com.*checkout.*/);
```

Comment:
Why this change, and do we still need a slash after the domain?

ie: `/paypal\.com\/.*checkout/`

Also, what is the URL that we need the `.*` in the middle?
I tried signing up for VPN (in Canada) using SubPlat+PayPal and it gave me something like this URL:
```
https://www.paypal.com/checkoutnow?sessionID=qwerty
```

Curious if the domain is very different in staging that we can't simplify the RegExp

Reply:
yeah for some reason the links are different on firefox vs chrome when testing. I believe for firefox, it has 'intent=checkout' instead of '/checkout' so this regex captures it.